plasma_framework/contracts/src/exits/payment/routers/PaymentInFlightExitRouterArgs.sol

```diff
@@ -15,6 +16,7 @@ library PaymentInFlightExitRouterArgs {
         bytes[] inputTxs;
         uint256[] inputUtxosPos;
         uint256[] inputUtxosTypes;
+        bytes[] outputGuardDataPreImages;
```

Comment:
Minor: I usually use `outputGuardPreimage` in other places. But I will leave this to you.